site/guide/4-requests.md

```diff
@@ -476,7 +476,7 @@ fn admin_panel_redirect() -> Redirect {
 
 The three routes above encode authentication _and_ authorization. The
 `admin_panel` route only succeeds if an administrator is logged in. Only then is
-the admin panel displayed. If the user is not an admin, the `AdminUser` route
```

Comment:
I think this actually meant to refer to the `AdminUser` *guard*. There is no `admin` route.

Reply:
Reading in context, I don't really like the existing wording, this wording, or the wording I just proposed.